install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -50,11 +50,18 @@ data:
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
         securityContext:
           capabilities:
+          {{- if .Values.global.proxy.privileged }}
             add:
-            - NET_ADMIN
-          {{ if .Values.global.proxy.privileged }}
+              - "NET_ADMIN"
           privileged: true
-          {{ end }}
+          {{- else }}
+            drop:
```

Comment:
Can we add one more option ? My concern is upgrade and safety - as a general rule we want new backward-incompatible behaviors to be protected by flags. 

For example: will core dump and debug continue to work ? We don't know what will break. 
I don't mind if in master we default to the secure config, or even remove the insecure option - but in 1.0 
I want to minimize upgrade risks.